CBLClient/Apps/CBLTestServer-Dotnet/TestServer/ListenerAuthenticatorMethods.cs

```diff
@@ -0,0 +1,32 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Net;
+using Couchbase.Lite.P2P;
+using Couchbase.Lite.Sync;
+using Couchbase.Lite.Testing;
+using JetBrains.Annotations;
+
+namespace Couchbase.Lite.Testing
+{
+    public class ListenerAuthenticatorMethods
+    {
+        public static void Create([NotNull] NameValueCollection args,
+                           [NotNull] IReadOnlyDictionary<string, object> postBody,
+                           [NotNull] HttpListenerResponse response)
+        {
+            var username = postBody["username"].ToString();
+            var password = postBody["password"].ToString();
```

Comment:
in a real app, password will not be sent as a plain text as a string over the wire. but I guess it should be fine with our test app. does ListenerPasswordAuthenticator takes encrypted value in its constructor?